app/controllers/backlogs_controller.rb

```diff
@@ -0,0 +1,25 @@
+class BacklogsController < ApplicationController
+
+  def index
+    @backlogs = Backlog.all
```

Comment:
Does this action ever get called? Is there a page that shows all the watchlists?